src/main/java/pl/equipRental/assets/AssetService.java

```diff
@@ -0,0 +1,21 @@
+package pl.equipRental.assets;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+@Service
+public class AssetService {
+    private final AssetRepository assetRepository;
+    private final AssetMapper assetMapper;
+
+    List<AssetDto> findAll() {
```

Comment:
Zastanowić się czy nie dodać tutaj Pageable, bo co się stanie kiedy będzie 1000 recordów w bazie to wtedy może być problem z wyświetlaniem tego na froncie

Reply:
Chyba że taki jest zamiar to jest git

Reply:
to zostawiamy na potem